src/client/firefox/events.js

```diff
@@ -53,6 +59,7 @@ async function paused(_: "paused", packet: PausedPacket) {
 
   if (why.type != "alreadyPaused") {
     const pause = createPause(packet, response);
+    flushSourceQueue();
```

Comment:
i think i'd prefer `sourceQueue.flush()` but that's minor 

`import * as sourceQueue from '../utils/source-queue'`